spec/views/user_index_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'Users Index Page', type: :feature do
+  before :each do
+    @user = User.create(name: 'John Snow', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                        bio: 'I am a human right activist.', posts_counter: 0)
+    @user2 = User.create(name: 'Magret Jones', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                         bio: 'I am a human right activist.', posts_counter: 0)
+  end
+  scenario 'displays all users username' do
+    visit '/'
+    expect(page).to have_content(@user.name)
+    expect(page).to have_content(@user2.name)
+  end
+
+  scenario 'displays all users profile photo' do
+    visit user_path(id: @user.id)
+    visit user_path(id: @user2.id)
+    find("img[src='https://unsplash.com/photos/F_-0BxGuVvo']")
+    find("img[src='https://unsplash.com/photos/F_-0BxGuVvo']")
+  end
+
+  scenario 'displays number of posts' do
+    visit '/'
+    expect(page).to have_content(@user.posts_counter)
+  end
+
+  it "redirected to that user's show page when I click on a user" do
+    visit '/'
+    click_link(@user.name)
+    expect(page).to have_content(@user.name)
+  end
+end
```

Comment:
- [ ] [REQUIRED] We have 4 tests failing for the `spec/requests/posts_spec.rb` and the `spec/requests/users_spec.rb:`. The `request` directory is not part of your file changes so I decided to attach them here.  Kindly fix the test errors before asking for another review.
<img width="817" alt="Screenshot 2022-11-01 at 20 31 14" src="https://user-images.githubusercontent.com/58692590/199325458-da3439dc-53f1-40c9-912f-fbc1518aedff.png">

<img width="870" alt="Screenshot 2022-11-01 at 20 37 11" src="https://user-images.githubusercontent.com/58692590/199325479-a7e33bbe-939b-4ce1-893e-d71873b10a6c.png">

<img width="868" alt="Screenshot 2022-11-01 at 20 36 26" src="https://user-images.githubusercontent.com/58692590/199325515-74a0289c-2760-4e6d-88ca-d6471b85d3e7.png">

<img width="807" alt="Screenshot 2022-11-01 at 20 34 52" src="https://user-images.githubusercontent.com/58692590/199325585-beea183c-fcd1-4627-9ab0-a40b14fd5e4d.png">

<img width="448" alt="Screenshot 2022-11-01 at 20 52 15" src="https://user-images.githubusercontent.com/58692590/199325969-0b05364f-c532-49c5-ad86-e1c7fd6c335a.png">